advanced-achievements-plugin/src/main/java/com/hm/achievement/config/ConfigurationParser.java

```diff
@@ -299,11 +311,12 @@ private void parseAchievements() throws PluginLoadError {
 		// Enumerate the achievements with multiple categories.
 		for (MultipleAchievements category : MultipleAchievements.values()) {
 			if (!disabledCategories.contains(category)) {
-				Set<String> subcategories = mainConfig.getConfigurationSection(category.toString()).getKeys(false);
-				if (subcategories.isEmpty()) {
+				if (getSectionKeys(category.toString()).isEmpty()) {
 					disabledCategories.add(category);
 					continue;
 				}
+
+				Set<String> subcategories = mainConfig.getConfigurationSection(category.toString()).getKeys(false);
```

Comment:
As we're calling `getSectionKeys` in the above conditional, could we not extract that to a local variable and reuse it here?

Reply:
Oops, I missed that!